pkgs/development/python-modules/quadprog/default.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+}:
+
+buildPythonPackage rec {
+  pname = "quadprog";
+  version = "0.1.11";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-jjIPn38UZhQeMX5hc26BCHIOatuuU/nWlg9ybv73iNk=";
+  };
+
+  propagatedBuildInputs = [
+    numpy
+  ];
+
+  meta = with lib; {
+    homepage = "https://github.com/quadprog/quadprog";
+    description = "Quadratic Programming Solver";
+    license = licenses.gpl2Only;
```

```suggestion
    license = licenses.gpl2Plus;
```

see https://github.com/quadprog/quadprog/blob/master/setup.py#L21